package.json

```diff
@@ -7,24 +7,26 @@
     "federalist": "export NODE_ENV=production && gulp buildAssets",
     "hugo": "export $(grep -v '^#' .env | xargs) && hugo serve --bind='0.0.0.0'",
     "lint:json": "find public/**/v1/json/index.html -print0 | xargs -0I {} jsonlint '{}'",
-    "lint:js": "eslint './themes/digital.gov/src/js/*.js'",
```

Comment:
@nick-mon1 ESlint was originally not linting any JS in ./config/gulp

Reply:
Could explain why it was so difficult to work with.

Reply:
I'm going to update this to lint the gulp files. 

I know I'm asking for trouble, but it only makes sense to lint all the JS into the project.